data/mods/CBM_Dungeon_Test/ls_mapgen_updates.json

```diff
@@ -0,0 +1,79 @@
+[
+  {
+    "type": "mapgen",
+    "update_mapgen_id": "UM_LS_Unlock",
+    "method": "json",
+    "object": { "translate_ter": [ { "from": "t_nl_door_l", "to": "t_nl_door_c", "x": [ 0, 23 ], "y": [ 0, 23 ] } ] }
```

Comment:
I'm removing translate_ter in favour of ter_furn_transform in #73437 and EOCs can call them directly iirc
I'm happy to change these over myself tho if this gets merged first